app/fullService/api/getAllAccounts.ts

```diff
@@ -1,13 +1,15 @@
-import type { Accounts } from '../../types/Account.d';
+import type { Accounts, Account, AccountsV2 } from '../../types/Account.d';
 import axiosFullService, { AxiosFullServiceResponse } from '../axiosFullService';
+import getAccount from './getAccount';
 
-const GET_ALL_ACCOUNTS_METHOD = 'get_all_accounts';
+const GET_ALL_ACCOUNTS_METHOD = 'get_accounts';
 
 type GetAllAccountsResult = Accounts;
 
-const getAllAccounts = async (): Promise<GetAllAccountsResult> => {
-  const { result, error }: AxiosFullServiceResponse<GetAllAccountsResult> = await axiosFullService(
-    GET_ALL_ACCOUNTS_METHOD
+export const getAllAccountsV2 = async (): Promise<AccountsV2> => {
+  const { result, error }: AxiosFullServiceResponse<AccountsV2> = await axiosFullService(
+    GET_ALL_ACCOUNTS_METHOD,
+    { offset: 0 }
```

Comment:
this should be the default value -- is it required here?

Reply:
oh, this was a hack that I need because of the way the `axiosFullService` handles params and because I added a third argument to `axiosFullService` to handle discerning between v1 and v2 api calls. because there is now a third optional argument to `axiosFullService` (v1 or v2), the second argument (the request params) can't be optional anymore. And some FS methods require there to be NO params field, while others require an empty params field if no params.

I'll fix this by changing adding `| null` to the type of the second argument to `axiosFullService` and using the presence of that arg to determine what to send to FS

Reply:
This is.. confusing haha thanks for parsing through it!